t/atnodes.t

```diff
@@ -56,6 +56,8 @@ OPTIONS:
     -v            Be verbose.
     -w            Prompt for password (used for login and sudo,
                   could be privided by SSH_BATCH_PASSWORD).
+    -W            Prompt for password (like -w, but used for sudo only,
+                  when -W is on, -w will be ignored).
```

Comment:
@agentzh Emm...dying loudly...how about "make sure not to use -W when -w is on, it will cause -w ignored". Excuse my poor English😂